Dockerfile

```diff
@@ -34,6 +34,8 @@ RUN rm -rf /var/lib/apt/lists/*
 # copy the aztarna files the FS and install it
 COPY . /root/aztarna
 # RUN cd /root/aztarna && git checkout ${AZTARNA_COMMIT} && python3.7 setup.py install
-RUN cd /root/aztarna && python3.7 setup.py install
+RUN cd /root/aztarna && python3 setup.py install
+RUN echo "source /opt/ros/dashing/setup.bash" >> /root/.bashrc
```

Comment:
Can I rework the docker part?

Reply:
I've pushed some changes related to the docker, let me know If you're happy with them.

Reply:
Docker output wit the latest changes:

```bash
lander@wks4:/tmp/aztarna(ros2_scanner)$ docker run  -it --rm --net=host aztarna -t ros2
[+] Host found in Domain ID 0
	Topics:
	Services:
		Service Name: /aztarna/describe_parameters 	|	 Service Type: ['rcl_interfaces/srv/DescribeParameters']
		Service Name: /aztarna/get_parameter_types 	|	 Service Type: ['rcl_interfaces/srv/GetParameterTypes']
		Service Name: /aztarna/get_parameters 	|	 Service Type: ['rcl_interfaces/srv/GetParameters']
		Service Name: /aztarna/list_parameters 	|	 Service Type: ['rcl_interfaces/srv/ListParameters']
		Service Name: /aztarna/set_parameters 	|	 Service Type: ['rcl_interfaces/srv/SetParameters']
		Service Name: /aztarna/set_parameters_atomically 	|	 Service Type: ['rcl_interfaces/srv/SetParametersAtomically']
	Nodes:
		Node Name: _ros2cli_daemon_0 	|	 Namespace: /
		Node Name: robot_state_publisher 	|	 Namespace: /
		Node Name: hros_cognition_kaladin 	|	 Namespace: /
		Node Name: minimal_action_server 	|	 Namespace: /
--------------------------------------------------------------------------------
[+] Host found in Domain ID 55
	Topics:
	Services:
		Service Name: /aztarna/describe_parameters 	|	 Service Type: ['rcl_interfaces/srv/DescribeParameters']
		Service Name: /aztarna/get_parameter_types 	|	 Service Type: ['rcl_interfaces/srv/GetParameterTypes']
		Service Name: /aztarna/get_parameters 	|	 Service Type: ['rcl_interfaces/srv/GetParameters']
		Service Name: /aztarna/list_parameters 	|	 Service Type: ['rcl_interfaces/srv/ListParameters']
		Service Name: /aztarna/set_parameters 	|	 Service Type: ['rcl_interfaces/srv/SetParameters']
		Service Name: /aztarna/set_parameters_atomically 	|	 Service Type: ['rcl_interfaces/srv/SetParametersAtomically']
	Nodes:
		Node Name: talker 	|	 Namespace: /
--------------------------------------------------------------------------------
```